docs/sni-limitation.rst

```diff
@@ -0,0 +1,35 @@
+.. _sni-limitation:
+
+Configure SNI TLS Load Balancer
+===============================
+
+Background
+----------
+
+Creating a new *TERMINATED_HTTPS* load balancer(LB for short) with new certificate(s) on OpenStack dashboard will trigger a new virtual server(VS for short) with new SSL profile(s) created on BIGIP side.
+
+It is possible to reuse existing certificates for creating new LB, which will trigger on BIGIP side, reusing existing SSL profiles for creating new VS.
+
+In summary:
+
+* Relation between certificate and SSL profile is one-to-one.
+* Certificate can associate to multiple LBs just as SSL profile can associate to multiple VSs on BIGIP side.
+
+
+SNI TLS Load Balancer
+---------------------
+
+The LB using multiple certificates called **SNI TLS Load Balancer**. Correspondingly, the triggered VS has multiple SSL profiles.
```

Comment:
```rst
A Neutron LBaaS load balancer that uses multiple certificates is known as an "SNI TLS Load Balancer". The corresponding BIG-IP virtual server also has multiple associated SSL profiles. One of the associated SSL profiles will be marked as the "Default SSL Profile for SNI".

.. caution::
   
   Setting more than one SSL profile as the "Default SSL Profile for SNI" will cause virtual server creation to fail. 

Because the OpenStack dashboard has no visibility into the state of the BIG-IP device, you need to check the BIG-IP system settings to find out which SSL profile is marked as the default for SNI.

Reply:
done